doc/source/cookbook.rst

```diff
@@ -5,24 +5,17 @@
 .. ipython:: python
    :suppress:
 
-   import pandas as pd
+   import datetime
+   import functools
+   import glob
+   import itertools
+   import os
    import numpy as np
+   import pandas as pd
    from pandas.compat import StringIO
-
-   import random
-   import os
-   import itertools
-   import functools
-   import datetime
-
    np.random.seed(123456)
```

Comment:
Hmm do we not need two blank lines separating this and imports? Surprised that wouldn't cause a flake8 error

Reply:
I think the two blank lines are a visual effect of github, I don't think there is any. Also could be that I'm too sleepy too, but I think is that github shows the space needed to show the lines removed in the left panel

Reply:
That's correct